docker/errors.py

```diff
@@ -76,3 +84,27 @@ def __str__(self):
 
 class NullResource(DockerException, ValueError):
     pass
+
+
+class ContainerError(Exception):
```

Comment:
Yes, good idea. I think a more general `DockerException` (or something) makes sense – it's technically not an error from the API.